cmd: add om-converter to convert stats.json to opemetrics

sambhav-jain-16 opened the pull request:
<what was there before: Previously, ...>
<why it needed to change: This was inadequate because ...>
<what you did about it: To address this, this patch ...>

sambhav-jain-16 committed:
cmd: add om-converter to convert stats.json to opemetrics

<what was there before: Previously, ...>
<why it needed to change: This was inadequate because ...>
<what you did about it: To address this, this patch ...>

(cherry picked from commit 4a382a036fed79eddbbc927c81d538da04d74379)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/141580)
<!-- Reviewable:end -->

sambhav-jain-16 committed:
fix isues

sambhav-jain-16 committed:
fix memory issues

sambhav-jain-16 committed:
add pprof

sambhav-jain-16 committed:
add pprof

sambhav-jain-16 committed:
fix statsexporter

sambhav-jain-16 committed:
fix statsexporter

sambhav-jain-16 committed:
add max warehouse again

sambhav-jain-16 committed:
add max warehouse again

sambhav-jain-16 committed:
add retry logic

sambhav-jain-16 committed:
fix buffer issues

sambhav-jain-16 committed:
fix buffer issues

sambhav-jain-16 committed:
fix new line  issues in statsexporter

sambhav-jain-16 committed:
fix tpccbench max issue

sambhav-jain-16 committed:
fix restore/tpce

sambhav-jain-16 committed:
remove cleaning logic

sambhav-jain-16 committed:
fix perturbation

sambhav-jain-16 committed:
decommissionBench: add post process step

1. Removes metrics duplication on pinnedNode and only use workloadNode
for metrics
2. Create a new file for decommission metrics rather than
appending to stats file
3. Adds post process step for decommission Bench
metrics

Epic: none

Release note: None
